### PR TITLE
Support for a non-clustering mode in tactile_merger

### PR DESCRIPTION
**Pull request by guihomework**
As discussed in #3 , in order to produce tactile PCLs easily for each taxels, a non-clustering option was added to the tactile_merger

### CODE REVIEW

**Comment by rhaschke** [comment]
I pushed some more commits directly.

File: tactile_merger/include/tactile_merger/merger.h [hunk 1]
```diff
@@ -42,8 +42,8 @@ class Merger
 	template <typename Iterator>
 	void update(const ros::Time &stamp, const std::string &channel,
 	            Iterator begin, Iterator end);
-	tactile_msgs::TactileContacts getAverageContacts();
-	tactile_msgs::TactileContacts getAllContacts();
+	tactile_msgs::TactileContacts getGroupContacts();
```
Comment (rhaschke) [inline tactile_merger/include/tactile_merger/merger.h]: Can we finish this? @guihomework, I know (from offline communication) that you want to keep _average_ as part of the function name. What about `getGroupAveragedContacts()`?

**Comment by guihomework** [comment]
Tests were done for this PR #5 on a live robot, but the tests with fake data done in #7  and concatenated in this image 

ContactStateIndividualTaxels_ShadowHand

showed inconsistencies that cannot easily be isolated to either the visualization or the new feature of the PR #5 
hence I would say #7 should be tackled first.

File: tactile_merger/include/tactile_merger/merger.h (hunk 1 above)
Comment (guihomework) [reply tactile_merger/include/tactile_merger/merger.h]: getGroupAveragedContacts is fine for me.

**Comment by rhaschke** [comment]
I will look into #7 (and the other open issues) later this evening.

**Comment by rhaschke** [comment]
As #7 is unrelated, I'm going to merge this.

**Pull request merged**